queries.sql

```diff
@@ -1,3 +1,10 @@
 /*Queries that provide answers to the questions from all projects.*/
 
-SELECT * from animals WHERE name = 'Luna';
+SELECT * FROM animals WHERE name like '%mon';
+SELECT name FROM animals WHERE date_of_birth > '2016-01-01' AND date_of_birth < '2019-01-01';
+SELECT name FROM animals WHERE neutered = true AND escape_attempts > 3;
+SELECT date_of_birth FROM animals WHERE name = 'Agumon' OR name = 'Pikachu';
+SELECT name,escape_attempts FROM animals WHERE  weight_kg > 10.5;
+SELECT * FROM animals WHERE neutered = true;
+SELECT * FROM animals WHERE name != 'Gabumon';
+ SELECT * FROM animals WHERE weight_kg <= 10.4 AND weight_kg <= 17.3;
```

Comment:
- [ ] Please here is `between...`: `weight_kg >= 10.4 AND weight_kg <= 17.3`

Reply:
fixed all that  thanks for the good review.